MainRepository/Program.cs

```diff
@@ -1,7 +1,68 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿// Program to find the possible combination of queen in the n*n chess board
+class NQueen {
+   static int Count;
+   //Function to check if two queens attack each other or not
+   // Unicode of queen-\u2655
+   static bool IsSafe (char[,] board, int r, int c) {
+      // Checking the column 
+      for (int i = 0; i < r; i++) {
+         if (board[i, c] == '\u2655')
+            return false;
+      }
+      // Checking the left diagonal 
+      for (int i = r, j = c; i >= 0 && j >= 0; i--, j--) {

```

Comment:
Same as previous if. Has single line in the body make it single line and remove the brackets.